src/services/dom/types.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+
+/**
+ * Copyright (c) Garuda Labs, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+export const HTTP_HEADERS = {
+  ACCEPT: 'Accept',
+  CONTENT_TYPE: 'Content-Type',
+  X_HYPERVIEW_VERSION: 'X-Hyperview-Version',
+  X_HYPERVIEW_DIMENSIONS: 'X-Hyperview-Dimensions',
+};
+
+export const HTTP_METHODS = {
+  GET: 'GET',
+  POST: 'POST',
```

Comment:
We technically also support `put`, `patch`, and `delete`, even though we don't use them from the IW backend:  https://hyperview.org/docs/reference_behavior_attributes#verb

Can you add some tests for those methods as well?

Reply:
As discussed on Slack, the use case for PUT/PATCH/DELETE does not exist with Hyperview, so we agreed to remove its support and update the doc. Done in 5619e0d